readme.txt

```diff
@@ -70,6 +70,11 @@ Your API Key can be found in your Mad Mimi account area. For more details [see t
 
 == Changelog ==
 
+= NEXT =
```

Comment:
@EvanHerman Love this idea but I don't think our Gruntfile supports this. You will have to add a new replacement pattern to the `replace:readme` task, and I would also suggest using `@NEXT` instead of `NEXT`.